src/config/services/index.md

```diff
@@ -32,3 +37,63 @@ Optionally, a service directory may contain:
 
 When a new service is created, a `supervise` folder will be automatically
 created on the first run.
+
+### Configuring services
+
+Most services can take configuration options set by a `conf` file in the service
+directory. This allows service customization without modifying the service
+directory provided by the relevant package.
+
+Check the service file for how to pass configuration parameters. Services
+usually expect `OPTS="--value ..."` in the `conf` file.
```

Comment:
"expect something like"? Are longopts usually used?

Reply:
I count seven services locally (out of a lot). Since it's the only "standard" mechanism, I'd like to have it. I think I can change it to something like "may use something like..." instead.

Reply:
Oh, sorry, i might have been unclear: OPTS should definitely be mentioned! i was just thinking that people may think longopts are always used for services; your suggested phrasing of "may use something like" is fine to deal with that. :-)